### ISSUE TITLE
is the example code correct? I got this error: PlatformException(not_available, No email client found, null, null)

### ISSUE THREAD

**Issue by lutang123** [sidlatau/flutter_email_sender#52]
Hi, I am not surely if I miss anything? All I did is running example code, and I typed my own email address as Recipient, but I couldn't send any email and got this error: 

PlatformException(not_available, No email client found, null, null)

Can anyone help me explain how to use this package? I followed the example code, but how does it work? does it make user send a message to my email (the recipient)?  but if I reply the message, where should i reply to? there is no sender's email. and anyway I can't make it work. And help is appreciated. Thank you!

**Comment by zjamshidi** [comment]
Do you have any email app on your test device? Have you signed in on the email app?

**Comment by DJSL1NKY** [comment]
I also get the same error.

I have Spark Mail on iOS as my default Mail App. I build my app with XCode onto my iPhone 11 Pro and getting the Error the same on the iOS Simulator. I downloaded the Apple Mail App changed the default app and installed my App on my Smartphone again, but it doens't help.

**Comment by AndriiHanets** [comment]
Same error with android emulator with installed and logged in gmail app

**Comment by Mosely** [comment]
I'm also having the same issue with Android Emulator and logged into the Gmail app.

**Comment by teekamsuthar** [comment]
I'm also getting the same issue with iOS. I have installed iOS default mail app and gmail too. Anybody knows a fix?

**Comment by Mosely** [comment]
I'm not sure about the iOS issue, but for Android, maybe it's related to app visibility changes in Android 11.  https://developer.android.com/training/basics/intents/package-visibility

I haven't tried it, yet, but what I'm going to do is change my app's targetSDK to 29 (Android 10) from 30 (Android 11) and see if that resolves my issue.

**Comment by Mosely** [comment]
To follow up on my previous post pertaining to Android 11 new privacy measures: yes, by changing the targetSDKversion to 29 and keeping the compileSDKversion at 30 in my app's build.gradle file, my app can find the Gmail app in the Android Emulator now.

**Comment by Dix206** [comment]
Instead of downgrading the targetSDKversion you can add the following to the manifest:
```
    <queries>
        <intent>
            <action android:name="android.intent.action.SEND" />
            <data android:mimeType="*/*" />
        </intent>
    </queries>
```
Also you need to update your gradle version in your build.gradle:
`classpath 'com.android.tools.build:gradle:4.0.1'`
and in your gradle/wrapper/gradle-wrapper.properties:
`distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip`

**Comment by 2shrestha22** [comment]
Here is the solution: https://devblogs.microsoft.com/xamarin/android-11-package-visibility/
To enable your application to get visbility into the packages you will need to add a list of queries into your AndroidManifest.xml.
```
<manifest package="com.mycompany.myapp">
  <queries>
    <intent>
      <action android:name="android.intent.action.SENDTO" />
      <data android:scheme="mailto" />
    </intent>
  </queries>
</manifest>
```

**Comment by sidlatau** [comment]
Fixed in v5.1.0
